xql/README.md

```diff
@@ -113,4 +113,56 @@ _Updated on 2024-01-08_
 7. [ ] **Aliases**: Add support to alias while querying.
 8. [ ] **Join Operations**: Support joining tables and apply query.
 9. [ ] **Nested Queries**: Add support to write nested queries.
-10. [ ] **Custom Aggregate Functions**: Support custom aggregate functions
\ No newline at end of file
+10. [ ] **Custom Aggregate Functions**: Support custom aggregate functions
+
+# `weather-lm` - Querying Xarray Datasets with SQL
+
+Querying weather data using Natural Language prompts. This uses a gemini (large language model from Google) to generate SQL like queries and `xql` to execute that query.
+> Refer to [Natural Language to Weather & Geospatial Data](https://docs.google.com/document/d/1zQ5jVIm29swVkv9nPM-vaDSatA6U3X3tv5EMATRAzU8/edit?usp=sharing&resourcekey=0-SIwltLLJD_MYkGH3H463WA) for more insights.
```

Comment:
This doc is not public, so let's remove this from here.

Reply:
Removed.